nautobot/core/templates/components/panel/body_content_objects_table.html

```diff
@@ -1,4 +1,7 @@
 {% load render_table from django_tables2 %}
-{% with table=body_content_table %}
-    {% render_table table 'inc/table.html' %}
-{% endwith %}
+<form id="object-table-form" method="post">
```

Comment:
Probably need a way to specify/override the `id` here as some views may have multiple ObjectsTablePanel in them?